README.md

```diff
@@ -1758,6 +1766,27 @@ extend-immutable-calls = ["fastapi.Depends", "fastapi.Query"]
 
 ---
 
+### `flake8-import-conventions`
+
+#### [`aliases`](#aliases)
```

Comment:
I think most similar to other setups would be to have `aliases` and `extend-aliases` (which defaults to `[]`, but is concatenated to `aliases`).

Reply:
@edgarrmondragon - I can tweak this and merge if you agree?

Reply:
Oh nm, you're all over it :)